main.tscn

```diff
@@ -3244,5 +3285,5 @@ layer_1/y_sort_origin = 0
 layer_1/z_index = 0
 layer_1/tile_data = PackedInt32Array(35, 393216, 18, 65571, 393216, 18, 131107, 393216, 18, 196643, 393216, 18, 262179, 393216, 18, 327715, 393216, 18, 393251, 393216, 18, 458787, 393216, 18, 524323, 393216, 18, 589859, 393216, 18, 655395, 393216, 18, 720931, 393216, 18, 786467, 393216, 18, 852003, 393216, 18, 917539, 393216, 18, 983075, 393216, 18, 1048611, 393216, 18, 1114147, 393216, 18, 1179683, 393216, 18, 1245219, 458752, 16, 0, 458752, 18, 65536, 458752, 18, 131072, 458752, 18, 196608, 458752, 18, 262144, 458752, 18, 327680, 458752, 18, 393216, 458752, 18, 458752, 458752, 18, 524288, 458752, 18, 589824, 458752, 18, 655360, 458752, 18, 720896, 458752, 18, 786432, 458752, 18, 851968, 458752, 18, 917504, 458752, 18, 983040, 458752, 18, 1048576, 458752, 18, 1114112, 458752, 18, 1179648, 458752, 18, 1245184, 458752, 17, 1, 262144, 14, 2, 327680, 14, 3, 327680, 14, 4, 327680, 14, 5, 327680, 14, 6, 327680, 14, 7, 327680, 14, 8, 327680, 14, 9, 327680, 14, 10, 524288, 11, 11, 327680, 14, 12, 327680, 14, 13, 327680, 14, 14, 327680, 14, 15, 327680, 14, 16, 327680, 14, 17, 327680, 14, 18, 327680, 14, 19, 327680, 14, 20, 327680, 14, 21, 327680, 14, 24, 327680, 14, 23, 327680, 14, 22, 327680, 14, 25, 327680, 14, 26, 327680, 14, 27, 327680, 14, 28, 327680, 14, 29, 327680, 14, 30, 327680, 14, 31, 327680, 14, 32, 327680, 14, 33, 327680, 14, 34, 393216, 14, 1245185, 327680, 18, 1245186, 327680, 18, 1245187, 327680, 18, 1245188, 327680, 18, 1245189, 327680, 18, 1245190, 327680, 18, 1245191, 327680, 18, 1245192, 327680, 18, 1245193, 327680, 18, 1245194, 327680, 18, 1245195, 327680, 18, 1245196, 327680, 18, 1245197, 327680, 18, 1245198, 327680, 18, 1245199, 327680, 18, 1245200, 327680, 18, 1245201, 327680, 18, 1245202, 327680, 18, 1245203, 327680, 18, 1245204, 327680, 18, 1245205, 327680, 18, 1245206, 327680, 18, 1245207, 327680, 18, 1245208, 327680, 18, 1245209, 327680, 18, 1245210, 327680, 18, 1245211, 327680, 18, 1245212, 327680, 18, 1245213, 327680, 18, 1245214, 327680, 18, 1245215, 327680, 18, 1245216, 327680, 18, 1245217, 327680, 18, 1245218, 327680, 18)
 
-[node name="Player" parent="TileMap" instance=ExtResource("2_rp60p")]
+[node name="Player" parent="." instance=ExtResource("2_rp60p")]
```

Comment:
Interesting, did we have the player as a child of the tile map?

Reply:
Yes. Accidentally I think!